app/messages/components/user-sider.tsx

```diff
@@ -0,0 +1,77 @@
+import AvatarList, { getColor } from "app/components/avatar-list"
+
+export type UserSiderProps = React.ComponentProps<typeof AvatarList>
+
+function UserList({ users }) {
+  return (
+    <div>
+      {users?.map((avatar, i) => {
+        return (
+          <div key={i} className="flex items-center first:pt-0 pt-8">
+            {avatar.pictureUrl ? (
+              <img
```

Comment:
All this we can get from the previous component, right? We may want to export smaller components from `avatar-list` such as `Avatar` to encapsulate this behavior